File: _daily_topic/math/conversionOfNumberSystems/171.cpp
#include "header.h"

class Solution {
public:
    int titleToNumber(string columnTitle) {
        long coe = 1;
        reverse(columnTitle.begin(), columnTitle.end());
        long ans = 0;
        for (auto c : columnTitle) {
            ans += coe * (c - 'A' + 1);
            coe *= 26;
        }
        return ans;
    }
};

File: _daily_topic/math/conversionOfNumberSystems/171.py
from typing import List
class Solution:
    def titleToNumber(self, columnTitle: str) -> int:
        mutiple = 1
        n = len(columnTitle)
        ans = 0
        for i in range(n):
            ans += mutiple * (columnTitle[i] - 'A' - 1)
            mutiple *= 26
        return ans